sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -880,30 +880,30 @@ object RapidsConf {
     .checkValues(RapidsReaderType.values.map(_.toString))
     .createWithDefault(RapidsReaderType.AUTO.toString)
 
-  val PARQUET_MULTITHREADED_COMBINE_THRESHOLD =
-    conf("spark.rapids.sql.format.parquet.multithreaded.combine.sizeBytes")
+  val READER_MULTITHREADED_COMBINE_THRESHOLD =
+    conf("spark.rapids.sql.reader.multithreaded.combine.sizeBytes")
       .doc("The target size in bytes to combine multiple small files together when using the " +
-        "MULTITHREADED parquet reader. With combine disabled, the MULTITHREADED reader reads the " +
-        "files in parallel and sends individual files down to the GPU, but that can be " +
-        "inefficient for small files. When combine is enabled, files that are ready within " +
-        "spark.rapids.sql.format.parquet.multithreaded.combine.waitTime together, up to this " +
+        "MULTITHREADED parquet or orc reader. With combine disabled, the MULTITHREADED reader " +
+        "reads the files in parallel and sends individual files down to the GPU, but that can " +
+        "be inefficient for small files. When combine is enabled, files that are ready within " +
+        "spark.rapids.sql.reader.multithreaded.combine.waitTime together, up to this " +
         "threshold size, are combined before sending down to GPU. This can be disabled by " +
         "setting it to 0. Note that combine also will not go over the " +
         "spark.rapids.sql.reader.batchSizeRows or spark.rapids.sql.reader.batchSizeBytes limits.")
       .bytesConf(ByteUnit.BYTE)
       .createWithDefault(64 * 1024 * 1024) // 64M
 
-  val PARQUET_MULTITHREADED_COMBINE_WAIT_TIME =
```

Comment:
you can't just remove these, you have to deprecate them in case someone is using them. The new ones should check if the old ones are set.  I probably messed up and should have started with them generic.

Reply:
Done